changeset:   3172:78300490d12c
tag:         tip
user:        David du Colombier <@intro@gmail.com>
date:        Sun Jun 13 21:32:22 2010 +0200
files:       src/cmd/auth/factotum/main.c
description:
factotum: revert commit 51363b8488f1


diff -r 194ca00ae29e -r 78300490d12c src/cmd/auth/factotum/main.c
--- a/src/cmd/auth/factotum/main.c	Wed May 19 18:27:05 2010 -0700
+++ b/src/cmd/auth/factotum/main.c	Sun Jun 13 21:32:22 2010 +0200
@@ -26,8 +26,6 @@
 	char *mtpt;
 	char err[ERRMAX];
 
-	rfork(RFNOTEG);
-
 /*	mtpt = "/mnt"; */
 	mtpt = nil;
 	owner = getuser();

